File: backend/src/main/java/com/interview/platform/mod09_interview_configuration/repository/InterviewConfigRepository.java
package com.interview.platform.mod09_interview_configuration.repository;

import com.interview.platform.mod09_interview_configuration.model.InterviewConfigEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.UUID;

public interface InterviewConfigRepository extends JpaRepository<InterviewConfigEntity, UUID> {
}

File: android-app/app/src/main/java/com/interview/platform/domain/repository/InterviewConfigRepository.kt
package com.interview.platform.domain.repository

import com.interview.platform.domain.model.mod09_interview_configuration.InterviewConfig
import com.interview.platform.data.model.Result
import kotlinx.coroutines.flow.Flow

interface InterviewConfigRepository {
    fun fetchAll(): Flow<Result<List<InterviewConfig>>>
    suspend fun syncData(): Result<Unit>
}
